### ISSUE TITLE
`closing-remarks` not showing on new computer?

### ISSUE THREAD

**Issue by aryairani** [refined-github/refined-github#7370]
Not sure if it's a local configuration error, a repo issue, or a regression?

Not sure if the personal access token is required, but I have added one and reloaded the page.
<img width="529" alt="image" src="https://github.com/refined-github/refined-github/assets/538571/8fbe68e0-aae8-4006-8506-dd30af7b9073">

**Comment by fregante** [comment]
Same situation as you, and I see it here:

https://togithub.com/refined-github/refined-github/pull/3689

<img width="518" alt="Screenshot 2024-05-23 at 15 01 22" src="https://github.com/refined-github/refined-github/assets/1402241/11979294-215e-4a17-a1c9-d8226b4b7d1d">

**Comment by aryairani** [comment]
Hm.. So how do I fix it? :)

I didn't know it was displayed at the bottom of the page, I thought it was at the top:

<img width="815" alt="image" src="https://github.com/refined-github/refined-github/assets/538571/a575537c-a1a3-43c6-b69f-876782a1f6ab">

Did it used to display something at the top of the page when it doesn't appear in any releases yet?

Or did something change about how pre-releases are shown?

I have this
<img width="861" alt="image" src="https://github.com/refined-github/refined-github/assets/538571/ea39e840-daba-4417-bbca-2ece61299da9">
<img width="445" alt="image" src="https://github.com/refined-github/refined-github/assets/538571/91ec2dfe-5243-4427-b506-648f3e754d86">

but seeing it at the bottom of a long page isn't very helpful; it should be mentioned at the top IMO.

**Comment by aryairani** [comment]
https://github.com/unisonweb/unison/pull/5002

**Comment by fregante** [comment]
The feature works correctly, read the message. The merge commit doesn't appear in any tags. This appears to be accurate. We just follow what GitHub tells us and there's no tag here, under the commit title: https://github.com/unisonweb/unison/commit/8a27961e9aa4fb1eea2d231a604cb722ee30dc6b

**Comment by aryairani** [comment]
It does show me a tag under the commit title at the link you provided:
<img width="519" alt="image" src="https://github.com/refined-github/refined-github/assets/538571/8b50a1eb-275f-4d6d-b3af-98b2f0ab0826">
the `trunk-build` tag

**Comment by fregante** [comment]
- https://github.com/refined-github/refined-github/issues/7206

**Comment by aryairani** [comment]
Sorry, I see the links but I'm not sure what you're telling me with them.

Do you mean that the behavior changed in February with #7243?

Looking at that PR I'm not sure what I'm meant to take away from it. My tag is not called `nightly`. I guess I should be looking at the regex and seeing that because my tag doesn't have a `.` in it, then it's also hidden, as of February of this year?

Or did you mean this?
> Also it might just be a Refined GitHub cache issue

**Comment by aryairani** [comment]
Sorry, I meant that you closed the issue without explanation. After you closed the issue without explanation, I asked for more explanation and you did respond, which I appreciate.

With regard to the template picker, it's not true that I disregarded anything. I chose "Ask a question or open a discussion", which doesn't have a template and doesn't ask for a URL.

<img width="1137" alt="image" src="https://github.com/refined-github/refined-github/assets/538571/2c761c0b-576a-4416-8c92-7b1157f4ef01">

Cheers
